service/awsconfig/v10/resource/s3bucket/current.go

```
@@ -15,31 +17,51 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 		return nil, microerror.Mask(err)
 	}
 
-	r.logger.LogCtx(ctx, "level", "debug", "message", "looking for the S3 bucket")
+	r.logger.LogCtx(ctx, "level", "debug", "message", "looking for the S3 buckets")
 
 	accountID, err := r.awsService.GetAccountID()
 	if err != nil {
 		return nil, microerror.Mask(err)
 	}
 
-	bucketName := key.BucketName(customObject, accountID)
-	headInput := &s3.HeadBucketInput{
-		Bucket: aws.String(bucketName),
-	}
-	_, err = r.clients.S3.HeadBucket(headInput)
-	if IsBucketNotFound(err) {
-		r.logger.LogCtx(ctx, "level", "debug", "message", "did not find the S3 bucket")
-		return BucketState{}, nil
-	}
-	if err != nil {
-		return BucketState{}, microerror.Mask(err)
+	bucketStateNames := []string{
+		key.TargetLogBucketName(customObject),
+		key.BucketName(customObject, accountID),
 	}
 
-	bucketState := BucketState{
-		Name: bucketName,
+	var currentBucketState []BucketState
+	for _, inputBucketName := range bucketStateNames {
+		inputBucket := BucketState{
+			Name: inputBucketName,
+		}
+
+		headInput := &s3.HeadBucketInput{
+			Bucket: aws.String(inputBucketName),
+		}
+		_, err = r.clients.S3.HeadBucket(headInput)
+		if IsBucketNotFound(err) {
+			r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("did not find the S3 bucket %q", inputBucketName))
+		} else if err != nil {
+			return nil, err
```

We should always mask errors. Please.

---

I missed this one. Thanks for catch